src/components/Popover/Popover.stories.tsx

```diff
@@ -147,3 +148,31 @@ export const LeftEnd: StoryObj<PopoverProps> = {
   },
   ...Default,
 };
+
+export const FocusClickableElement: StoryObj<PopoverProps> = {
+  play: async ({ canvasElement }) => {
+    // We want to test visual regression for the Popover.Content as well as the button,
+    // but don't want the drawer open initally outside Chromatic.
+    if (isChromatic()) {
+      const canvas = within(canvasElement);
+      const filtersTrigger = await canvas.findByRole('button');
+      userEvent.click(filtersTrigger);
+    }
+  },
+  render: (args) => {
+    return (
+      <Popover {...args}>
+        <Popover.Button as={Button} data-testid="popover-trigger-button">
+          Open Popover
+        </Popover.Button>
+        <Popover.Content data-testid="popover-content" focus>
```

Comment:
I may be missing something, why does the content need `focus` set?

Reply:
[`focus`](https://headlessui.com/react/popover#popover-panel) enables the behavior where it will focus on the element once the popover is generated